osu.Game/Graphics/OsuFont.cs

```diff
@@ -136,10 +136,5 @@ public enum FontWeight
         /// Equivalent to weight 700.
         /// </summary>
         Bold = 700,
-
-        /// <summary>
-        /// Equivalent to weight 900.
-        /// </summary>
-        Black = 900
```

Comment:
not sure about removing here, but i guess it's okay for now?

Reply:
would've preferred to keep it for reference purposes too.